spec/js/views/todo_view_spec.js

```diff
@@ -40,4 +39,33 @@ define([
       expect(instance.$el.find('.delete').length).toBe(expectedLength);
     });
   });
+
+  describe('Check item', function () {
+    beforeEach(function () {
+      subject = function () {
+        instance.render();
```

Comment:
You can move `instance.render()` outside the subject since it is actually a setup.